Resources/public/js/views/fields/ez-date-view.js

```diff
@@ -48,6 +48,20 @@ YUI.add('ez-date-view', function (Y) {
             }
             return '';
         },
+
+        /**
+         * Formats the date part of the date object
+         *
+         * @method _formatDate
+         * @protected
+         * @param {Date} date
+         * @return String
+         */
+        _formatDate: function (date) {
+            return date.toLocaleDateString(
```

Comment:
It's better readable when it looks like this:`return date.toLocaleDateString(undefined, {timeZone: 'UTC'});`